### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by ahmadjana**
this branch for our open mic sessions:
It should contains the schedule and contents of the seminars

### CODE REVIEW

**Comment by ahmadjana** [comment]
@blcham

**Comment by ledsoft** [comment]
Does it make sense to have the open mic schedule publicly visible, when the sessions are internal to KBSS? Or is there a plan to make the sessions public?

**Comment by blcham** [comment]
> Does it make sense to have the open mic schedule publicly visible, when the sessions are internal to KBSS? Or is there a plan to make the sessions public?

Yes, this was the idea, see #7. I would like to publish some of the presentations and possibly invite somebody to join our presentation online.

**Review by blcham — COMMENTED** [review]
1) I don't like that we are sharing our kbss zoom channel
   - I suggest to create google meeting room for that with KBSS members beeing administrators. Thus we will be able control who comes into the room. 
2) @ledsoft  did not agree yet with the publishment of recordings ... so do you ?!
3) WRT, idea of makeing recording public ... if you think that it should not be public by default (we will of course put there only recordings that are good enough), we can do that on request (so always approve somebody who wil ask to see it -- e.g. by using google drive request permissions)

@ledsoft @ahmadjana

**Comment by MichalMed** [comment]
Good thing is to have some static page with programme and link it every time like three days before.

File: _pages/open_mic.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ***KBSS Open Mic Session***
+## [Domain-specific Semantic Web-based Information Systems - Any Progress?](https://github.com/ahmadjana/kbss-website/blob/patch-1/_pages/seminar1.md)
```
Comment (MichalMed) [inline _pages/open_mic.md:2]: use relative links

File: _pages/seminar1.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ***[Domain-specific Semantic Web-based Information Systems - Any Progress?](https://www.researchgate.net/publication/357127663_A_Combined_Security_Ontology_based_on_the_Unified_Foundational_Ontology)***
```
Comment (MichalMed) [inline _pages/seminar1.md:1]: missing front matter

File: _pages/seminar1.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+### *Date and Time:*   11.03.2022: 10:30 am
```
Comment (MichalMed) [inline _pages/seminar1.md:2]: this is header

File: _pages/seminar2.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# ***[Collecting meaningful data: an iterative approach](https://drive.google.com/file/d/1SHU5Th5iSMbtkgR06W6w7xTzc-5pk5-k/view)***
```
Comment (MichalMed) [inline _pages/seminar2.md:1]: same as seminar 1

**Review by MichalMed — CHANGES_REQUESTED** [review]
I'd suggest to create posts according to the one already created (JTBD). Also I think it is time to announce next weeks open mic session as a post.

File: _pages/open_mic.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+  - 11.03.2022: 10:30 am on  [zoom channel](https://feectu.zoom.us/j/668775388)
+## [Collecting meaningful data: an iterative approach](https://github.com/ahmadjana/kbss-website/blob/patch-1/_pages/seminar2.md)
+- 01.04.2022: 10:30 am on  [zoom channel](https://feectu.zoom.us/j/668775388)
+## [Jobs o be Done](https://kbss-cvut.github.io/kbss-website/jtbd-in-openmic)
```
[the remark this answers is not in the document]
Comment (ahmadjana) [reply _pages/open_mic.md]: @MichalMed  what do you mean (use the relative link?)

[the remark this answers is not in the document]
Comment (MichalMed) [reply _pages/open_mic.md]: this link points to the exact position in the internet, into your repository at github. But the webapp can be built anywhere, so use the relative link to the place where the source is located in the project, e.g. {{ "seminar2.md" | relative_link }}. MOre in the jekyll and minimal mistakes documentation

**Pull request closed without merging**